pom.xml

```diff
@@ -4,7 +4,14 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
-    <groupId>org.example</groupId>
+    <parent>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-starter-parent</artifactId>
+        <version>2.7.9</version>
+        <relativePath/> <!-- lookup parent from repository -->
+    </parent>
+
+    <groupId>com.epam.learning.backendservices.rest</groupId>
     <artifactId>REST_API</artifactId>
     <packaging>pom</packaging>
     <version>1.0-SNAPSHOT</version>
```

Comment:
Optional: to make version maitenance convenient you can do like this:
pom.xml
```xml
<groupId>com.epam.learning.backendservices.rest</groupId>
<artifactId>REST_API</artifactId>
<packaging>pom</packaging>
<version>${revision}</version>

<properties>
  <revision>1.0-SNAPSHOT</revision>
</properties>
```
jmp-cloud-service-impl/pom.xml
```xml
<parent>
  <groupId>com.epam.learning.backendservices.rest</groupId>
  <artifactId>REST_API</artifactId>
  <version>${revision}</version>
</parent>
```

Reply:
Done